Automattic/simplenote-macos
# Use Sparkle Pod instead of embedded framework

I was getting a bunch of strange compile errors related to Sparkle. We can use a `Sparkle` pod now, so I've updated the app to use that and it fixes the compile errors.

**To Test**
* `pod install`
* Add `USE_HOCKEY=1` in the preprocessor macros for `Debug` (this will include sparkle in the app delegate):
<img width="712" alt="screen shot 2017-11-13 at 11 02 33 am" src="https://user-images.githubusercontent.com/789137/32743757-31e748e4-c862-11e7-9965-9320b0fce18f.png">

* Run the app, it should compile!
